test/test_dao.py

```diff
@@ -1,3 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
This dao is for WaterML 1.0. Which is not our main focus currently when trying to solve this unicode problems. Though thanks for adding the similar test in anyways 😄

Reply:
I picked your commit :smile: